spec/js_routes/amd_compatibility_spec.rb

```diff
@@ -27,12 +27,8 @@
     evaljs(JsRoutes.generate({}))
   end
 
-  it "should working from global scope" do
```

Comment:
This test should work for cases, where not "build system" (include file by `<script>` tag should be also valid)

Reply:
This test doesn't work right now because `window.define` is defined which prevents the global from being created when using UMD modules. I can verify that this still works in another test